arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/Ior.kt

```diff
@@ -428,6 +428,15 @@ sealed class Ior<out A, out B> {
       { a, b -> fa(a).zip(fb(b)) { aa, c -> Both(aa, c) } }
     )
 
+  inline fun <C, D> bitraverseOption(
+    fa: (A) -> Option<C>,
+    fb: (B) -> Option<D>
+  ): Option<Ior<C, D>> =
+    bitraverseEither(
+      { fa(it).toEither { Unit } },
+      { fb(it).toEither { Unit } }
+    ).orNone()
```

Comment:
```suggestion
    fold(
      { fa(it).map { Left(it) } },
      { fb(it).map { Right(it) } },
      { a, b -> fa(a).zip(fb(b)) { aa, c -> Both(aa, c) } }
    )
```

Nit: a direct implementation reads better IMO, and is (slightly) more performant.